### ISSUE TITLE
Feature request: export dummy characters & import eve format

### ISSUE THREAD

**Issue by amkoc** [pyfa-org/Pyfa#717]
It'd be nice if we could export characters created in the Character Editor to be imported in other people's pyfa installs - corps could have their skill plans as characters and ensure the doctrine ships fit it.

Also to that end, it'd be nice if we could simply copy/paste skills into the Character Editor in the same list-of-skill-levels format EVE itself accepts for the skill queue.

**Comment by blitzmann** [comment]
I would prefer people use evemon character sheets for porting around characters. pyfa does accept those as an import option. However, this copy and paste skill list... can you explain that a bit more? I did not know EVE allowed for that. Please post the format and I can look into supporting a simple text import/export for that. :)

**Comment by amkoc** [comment]
> However, this copy and paste skill list... can you explain that a bit more? I did not know EVE allowed for that. 

Yep, some time ago eve added the option to import a list of skills into eve with a button click. It's just the skill you want to train and the level in roman numerals.

Here's a chunk of my corp's newbie skillplan as an example

`Minmatar Frigate I
Minmatar Frigate II
Gallente Frigate I
Gallente Frigate II
Caldari Frigate I
Caldari Frigate II
Small Hybrid Turret I
Small Projectile Turret I
Sensor Linking I
Target Painting I
Cloaking I
Minmatar Frigate III
Propulsion Jamming II
Minmatar Destroyer I
Minmatar Destroyer II
Infomorph Psychology I
Missile Launcher Operation II
Light Missiles I
Light Missiles II
Gallente Frigate III
Gallente Destroyer I
Gallente Destroyer II
Hull Upgrades III
`

Just open up the skill queue window in eve, and hit the 'copy from clipboard' and it'll put the list together, with your choice of replacing all the skills with the plan or just adding them to the queue.

Since this was added, many skill plans can be found in that format as it is convenient to import into EVE.
